locations/storefinders/wakefern.py

```diff
@@ -0,0 +1,42 @@
+import json
+import re
+
+from locations.hours import OpeningHours
+from locations.json_blob_spider import JSONBlobSpider
+from locations.pipelines.address_clean_up import merge_address_lines
+
+
+class WakefernSpider(JSONBlobSpider):
```

Comment:
Can you merge this with the "Storefrontgateway" store locator, the telltale is the presence of "mi9-storefront" in some of these pages.
Your version works off the frontend js; but there should also be an API endpoint like `https://storefrontgateway.saveonfoods.com/api/stores` on a lot of these.